library/Zend/Controller/Action/Helper/ViewRenderer.php

```diff
@@ -841,14 +841,20 @@ protected function _translateSpec(array $vars = array())
         $inflector  = $this->getInflector();
         $request    = $this->getRequest();
         $dispatcher = $this->getFrontController()->getDispatcher();
-        $module     = $dispatcher->formatModuleName($request->getModuleName());
-        $controller = $dispatcher->formatControllerName(
-            $request->getControllerName()
-        );
+
+        // Format module name
+        $module = $dispatcher->formatModuleName($request->getModuleName());
+
+        // Format controller name
+        $filter     = new Zend_Filter_Word_CamelCaseToDash();
```

Comment:
I think we need to `require_once 'Zend/Filter/Word/CamelCaseToDash.php` somewhere.